app/settings/urls.py

```diff
@@ -76,16 +56,30 @@
     path(
         "api/v1/is-authorized/",
         IsAuthorizedView.as_view(),
-        name="is-authorized",
+        name="is-authorized-v1",
+    ),
+    path(
+        "api/v2/is-authorized/",
+        IsAuthorizedView.as_view(),
+        name="is-authorized-v2",
```

Comment:
Ik vond je refactor van de routing naar de router.py van elke app wel een mooie toevoeging, is er een reden dat deze hier nog staan of kunnen die ook naar de router.py van de users app?

Reply:
Ja, ik zal die ook nog even toevoegen inderdaad, dan is de namespace compleet